ush/valid_param_vals.sh

```diff
@@ -22,6 +22,7 @@ valid_vals_PREDEF_GRID_NAME=( \
 "GSD_HRRR_AK_50km" \
 "GSD_RAP13km" \
 "RRFS_NA_3km" \
+"RRFS_firewx_1.5km" \
```

Comment:
@BenjaminBlake-NOAA, as we can see above, all the domain (grid) names are in upper case. What do you think about changing the domain name to `RRFS_FIREWX_1.5km`?

Reply:
Sure!  I'll get that changed.